manifests/init.pp

```diff
@@ -214,25 +236,21 @@
       $_arg_workers = ''
     }
 
-    $cmd = '/usr/bin/createrepo'
+    $cmd = "/usr/bin/${createrepo_package}"
```

Comment:
This is a bit hairy as there's a requirement that the main binary has the same name as the package. I would like to see the $cmd parameterized with the default of `/usr/bin/createrepo` and giving users the option of replacing it with whatever they like.

Reply:
I agree, I'll add an extra parameter.